tests/test_auth.py

```diff
@@ -0,0 +1,51 @@
+import contextlib
```

Comment:
This way too complex set up for simple test. Let's just make simple get request and check that api returns correct error. And I don't think we need that many fixtures, and you can you token_factory to generate token and factory for generating user_data.